backend/src/main/java/com/cic/service/MatchingService.java

```diff
@@ -0,0 +1,27 @@
+package com.cic.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import com.cic.openapi.model.Match;
+import com.cic.openapi.model.Person;
+import com.cic.service.matching.MatchingAlgorithm;
+
+public class MatchingService {
+
+  private final MatchingAlgorithm matcher;
+
+  public MatchingService(MatchingAlgorithm matcher) {
+    this.matcher = matcher;
+  }
+
+  public List<Match> returnMatches(Set<Person> mentees, Set<Person> mentors) {
+    List<Match> matches = new ArrayList<>();
+
+    // for (int i = 0; i < mentees.length; i++) {

```

Comment:
Is this commented out code fine to be removed?

Reply:
No this is meant to be implemented in the next pr just putting some filler for now